File: Programming-languages-and-laboratory/Java/Exercises/31_compito_java_2019_07_05/Auto.java
import java.util.*;

public class Auto extends Veicolo {
    private int cilindrata;
    private double bagagliaio;
    private String categoria;

    public Auto(int codice, String targa, String modello, String marca, double costo, int cilindrata, double bagagliaio, String categoria) {
        super(codice, targa, modello, marca, costo);
        this.cilindrata = cilindrata;
        this.bagagliaio = bagagliaio;
        this.categoria = categoria;
    }

    public String toString() {
        return "automobile\t" + super.toString() + bagagliaio + "\t" + cilindrata + "\t" + categoria + "\t-\t-\t-";
    }
}

File: II - Linguaggi di programmazione e laboratorio/Esame/Python/40_compito_python_20220620/Auto.py
from Veicolo import Veicolo


class Auto(Veicolo):

    def __init__(self, codice, marca, modello, targa, costo, cilindrata, bagagliaio):
        super().__init__(codice, marca, modello, targa, costo)

        self._cilindrata = cilindrata
        self._bagagliaio = bagagliaio

    def __str__(self):
        return "automobile\t" + super().__str__() + str(self._cilindrata) + "\t" + str(self._bagagliaio) + "\t-\t-\t-"
